mautic/mautic
# Anonymize IP-Addresses

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | yes
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/3445#issuecomment-290732453
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Added new option to config to Anonymize IP-Addresses (192.168.0.***) based on this comment discussion https://github.com/mautic/mautic/issues/3445  and comment https://github.com/mautic/mautic/issues/3445#issuecomment-290732453

![image](https://user-images.githubusercontent.com/462477/40480800-d38256ba-5f4f-11e8-9ec2-b5ed78e40131.png)

#### Steps to test this PR:
1. Enable new option "Anonymize IP" in Configurations > Tracking settings
2. Go to page with tracking code in inkognito
3. See if your IP address is anonymized, but tracking still works
